Remove blocks with uninitialized, and apparently otherwise unused variables
const int tbg_version_id is needed during compilation, will try to set to 0

Index: src/core/Version.cpp.in
--- src/core/Version.cpp.in.orig
+++ src/core/Version.cpp.in
@@ -25,14 +25,10 @@
  * For available variables see cmake/VersionString.cmake.
  */
 
-#if ${VERSION_COUNT} != 4 && ${VERSION_COUNT} != 5
-#error "Configure error - the VERSION file should specify three or five non-empty lines!"
-#endif
-
 const std::string arx_version = "${VERSION_0}${GIT_SUFFIX_5}";
 const std::string arx_release_codename = "${VERSION_4}";
 
 // TODO move this to an external file once we ship our own resources
 const std::string arx_credits = "${AUTHORS}";
 
-const int tbg_version_id = ${VERSION_2};
+const int tbg_version_id = 0;
